src/Resources/command/LconfigCommand.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import logging
+import re
+import sys
+
+from .. import utils
+from ..foundation.command.Command import Command
+from ..manager.ManagerProxy import ManagerProxy
+from ..model.Lab import Lab
+from ..strings import strings, wiki_description
+
+
+class LconfigCommand(Command):
+    __slots__ = []
+
+    def __init__(self):
+        Command.__init__(self)
+
+        parser = argparse.ArgumentParser(
+            prog='kathara lconfig',
+            description=strings['lconfig'],
+            epilog=wiki_description,
+            add_help=False
+        )
+
+        parser.add_argument(
+            '-h', '--help',
+            action='help',
+            default=argparse.SUPPRESS,
+            help='Show an help message and exit.'
+        )
+        parser.add_argument(
+            '-d', '--directory',
+            metavar='LAB_PATH',
+            required=False,
+            help='Path of the lab to configure, if not specified the current path is used'
+        )
+        parser.add_argument(
+            '-n', '--name',
+            metavar='DEVICE_NAME',
+            required=True,
+            help='Name of the machine to be connected on desired collision domains.'
+        )
+        parser.add_argument(
+            '--eth',
+            dest='eths',
+            metavar='N:CD',
```

Comment:
Change `metavar` in `CD` instead of `N:CD`, this should be also done in `vconfig`.